src/components/address-book/AddressBookHeader/index.tsx

```diff
@@ -11,11 +15,46 @@ import ImportIcon from '@/public/images/address-book/import.svg'
 import ExportIcon from '@/public/images/address-book/export.svg'
 import AddCircleIcon from '@/public/images/common/add-outlined.svg'
 
+const HeaderButton = ({
+  icon,
+  onClick,
+  disabled,
+  children,
+}: {
+  icon: ElementType
+  onClick: () => void
+  disabled?: boolean
+  children: string
+}): ReactElement => {
+  const theme = useTheme()
+  const isMobile = useMediaQuery(theme.breakpoints.down('md'))
```

Comment:
Can we avoid this? E.g. by using vh/vw units or a media query in CSS?

Reply:
I'd sooner reference the breakpoints via the `theme` as it matches the `Grid` logic below.

Reply:
How many pixels is `md`? We use `(max-width: 600px)` elsewhere in the app for mobile.

Reply:
I just don't see the point doing JS detection just to make an icon smaller.

The tooltip can be shown on desktop as well, no harm in that.

Reply:
You can't even see the tooltip, you have to tap and hold on the button to make it appear.

Reply:
That toolbar has usability issues on mobile, but that's beside the point.
Please don't use JS for responsiveness is my point.